test/views/wallet/TransactionView/TransactionView.test.tsx

```diff
@@ -0,0 +1,63 @@
+import React from 'react';
+
+import { screen } from '@testing-library/react';
+import userEvent from '@testing-library/user-event';
+
+import useMobileCoinD from '../../../../app/hooks/useMobileCoinD';
+import { TransactionView } from '../../../../app/views/wallet';
+import renderSnapshot from '../../../renderSnapshot';
+
+jest.mock('../../../../app/hooks/useMobileCoinD');
+
+function setupComponent() {
+  const mockUseMobileCoinD = useMobileCoinD as jest.MockedFunction<
+    typeof useMobileCoinD
+  >;
+
+  // @ts-ignore mock
+  mockUseMobileCoinD.mockImplementation(() => {
+    return {
+      accountName: 'account name',
+      b58Code: 'b58 code',
+      mobUrl: 'string',
+    };
+  });
+
+  const { asFragment } = renderSnapshot(
+    <TransactionView />,
+  );
+
+  const sendQuery = screen.queryByText(/Please enter the amount of MOB you want to send and the public address of the recipient./i);
+  const receiveQuery = screen.queryByText(/To receive MOB, you must share your public address code to the sender./i);
```

Comment:
Just a question: Is the final /.../i actually needed?

Reply:
It wasn't, was still beginning with testing library so tried querying different ways.